proc_status.go

```diff
@@ -76,6 +77,9 @@ type ProcStatus struct {
 	UIDs [4]string
 	// GIDs of the process (Real, effective, saved set, and filesystem GIDs)
 	GIDs [4]string
+
+	// CPUSET: list of cpu cores, allowed to run process
```

Comment:
Minor nit Go documentation comments need to match.

```suggestion
	// CpusAllowedList: List of cpu cores processes are allowed to run on.
```